src/status_im/core.cljs

```diff
@@ -40,7 +40,7 @@
 
 (def adjust-resize 16)
 
-(defn is-hermes
+(defn hermes?
```

Comment:
Again, not related, couldn't resist, renamed this function because this is not JS